woocommerce/woocommerce
# change extension on scaffolding sample scripts

### Changes proposed in this Pull Request:

This PR addresses a circular reference error in `nx` by renaming two sample scripts used by the e2e tests installer
```
Switched to branch 'add/config-default-values'
Your branch is up to date with 'origin/add/config-default-values'.
$ pnpm nx build woocommerce
 
>  NX   ERROR  Could not execute woocommerce:build because it has a circular dependency

  woocommerce:build --> @woocommerce/e2e-core-tests:build --> @woocommerce/e2e-environment:build --> @woocommerce/e2e-utils:build --> @woocommerce/e2e-environment:build
```

Yesterday, #31428 was reverted after receiving report of the above issue. While reverting the change eliminates the error, nx was finding the circular reference in a sample script that is never executed. The error didn't occur when using the `pnpm` commands which allowed the PR to pass CI and reviewer testing.


### How to test the changes in this Pull Request:

1. Run `pnpm i` in the `woocommerce/plugins` folder
2. Rename the `tests` folder to `tests-x`
3. Run `pnpx wc-e2e install`
4. Confirm that `tests/e2e/config/jest.config.js` and `tests/e2e/config/jest.setup.js`
5. Delete the `tests` folder and rename `tests-x` to `tests`
